docs/docs/cloud/how-tos/human_in_the_loop_time_travel.md

```diff
@@ -155,17 +156,23 @@ Output:
 
 
 
-To rerun from a state, we need to pass in the `checkpoint_id` into the config of the run like follows:
+To rerun from a state, we need first issue an empty update to the thread state. Then we need to pass in the resulting `checkpoint_id` as follows:
 
 === "Python"
 
     ```python
+    state_to_replay = states[2]
+    updated_config = await client.threads.update_state(
+        thread["thread_id"],
+        {"messages": []},
+        checkpoint_id=state_to_replay["checkpoint_id"]
+    )
     async for chunk in client.runs.stream(
         thread["thread_id"],
         assistant_id, # graph_id
         input=None,
         stream_mode="updates",
-        config={"configurable": {"checkpoint_id": state_to_replay['checkpoint_id']}}
+        checkpoint_id=updated_config["configurable"]["checkpoint_id"]
```

Comment:
actually this should read from just checkpoint_id, see https://github.com/langchain-ai/langgraph-api/pull/283/files#diff-f47d7b40f2c02cc4ba7b4f25aa6313c2d132d13fffe65e30d15bc2a969df94f8R435

Reply:
yup good point, need to update